resources/assets/js/app/controllers/ShowUserVotes.js

```diff
@@ -0,0 +1,26 @@
+var Marionette = require('backbone.marionette');
+var app = require('../instances/appInstance');
+var voteLayout = require('../views/votes/MyVoteLayout');
+var myVoteCollection = require('../collections/MyVoteCollection');
+var myVoteModel = require('../models/MyVoteModel');
+var myVoteDetailView = require('../views/votes/MyVoteDetail');
+
+module.exports = Marionette.Object.extend({
+    index: function () {
+        var myVoteCollection = new myVoteCollection();
+        myVoteCollection.fetch();
+        app.render(new voteLayout({collection: myVoteCollection}));
+    },
+    show: function (id) {
+        var myVoteModel = new myVoteModel({id: id});
+        myVoteModel.fetch({
+            success: function () {
+                app.render(new myVoteDetailView({model: myVoteModel}));
```

Comment:
Don't need to use synchronous render. Just set model to a View and it will render all necessary information when it will be fetched. If not, then add to a View model event 'sync'